### PR TITLE
AAE-729 fix CatchEventXMLConverter, ThrowEventXMLConverter, test

### PR DESCRIPTION
**Pull request by CTI777**
Add to the sections for definition of IntermediateThrowEvent, IntermediateCatchEvent incoming and outgoing elements

### CODE REVIEW

File: activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/CatchEventXMLConverter.java [hunk 1]
```diff
@@ -18,14 +18,16 @@
...
 import org.activiti.bpmn.model.IntermediateCatchEvent;
 
 /**
 
  */
 public class CatchEventXMLConverter extends BaseBpmnXMLConverter {
 
-  public Class<? extends BaseElement> getBpmnElementType() {
+  @Override
+public Class<? extends BaseElement> getBpmnElementType() {
```
Comment (igdianov) [inline activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/CatchEventXMLConverter.java]: add indent

File: activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/ThrowEventXMLConverter.java [hunk 2]
```diff
@@ -18,14 +18,16 @@
...
 import org.activiti.bpmn.model.ThrowEvent;
 
 /**
 
  */
 public class ThrowEventXMLConverter extends BaseBpmnXMLConverter {
 
-  public Class<? extends BaseElement> getBpmnElementType() {
+  @Override
+public Class<? extends BaseElement> getBpmnElementType() {
```
Comment (igdianov) [inline activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/ThrowEventXMLConverter.java]: please, add indentation.

**Comment by CTI777** [comment]
Fixed

**Comment by erdemedeiros** [comment]
Ref https://github.com/Activiti/Activiti/issues/3008

**Pull request merged**